test/integration/abm_gce_bash/abm_gce_bash_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package abmeditor
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestABMBash(t *testing.T) {
+	abm := tft.NewTFBlueprintTest(t)
```

Comment:
You may have to pass in a mock temp dir. It usually tries to detect the TF dir but in this case there is no TF directory.
```suggestion
	abm := tft.NewTFBlueprintTest(t, tft.WithTFDir(t.TempDir()))
```